python/tests/test_panda.py

```diff
@@ -45,7 +45,8 @@ def test_forward_kinematics(self):
             [0, -1, 0],
             [0, 0, -1]
         ])
-        expected_sT7 = Pose3(sR7, Point3(0.129315, 0.00227401, 0.88387))
+        # regression
+        expected_sT7 = Pose3(sR7, Point3(0.139535, 0.004392, 0.921429))
```

Comment:
same comment about documenting regression reasoning (via reply to these comments)